develop.sh

```diff
@@ -37,6 +37,9 @@ function invoke_docker_compose_spark {
 }
 
 function format_namenode {
+    invoke_docker_compose_spark down
```

Comment:
ugh, this no longer works because I updated invoke_docker_compose_spark to use `exec` in #994. Will have to work out the best way of doing it

Reply:
this is fixed, I use a subshell, and document the syntax